test/extended/testdata/builds/test-build-app-pipeline-upstream.git/objects/8f/cda77484140980cb3f580797b8efb86b7e2a70

```diff
@@ -0,0 +1,2 @@
+x��K
+�0@]�s����Ā�W�3�E�S����\�����)8L�"��ыB]���K�D�MR��S�1�3��mL�����[�Sx+z�����<��G�6DX0 ��?'T��3moj�h�FK
```

Comment:
Can the repository be fetched from remote upon test's initialization instead of making it a kind of submodule?

Reply:
yes, it can. The previous commit in this PR is done that way but the available repo is unnecessarily large (as is the case for good number of extended tests and partially a reason, why they take so long) and I thought this might be a good way to speed up the tests - have minimal repos locally.

Reply:
I see the benefit although a readable source code is still preferrable over the binary blobs. In case we don't need to perform any remote calls as a part of the test, would something along the following pseudo-bash lines work?

```sh
mkdir testrepo && cd testrepo
touch sourcefile
git init .
git add *
git commit --autor='testrobot <do@not.blame.me>'  -m "nothing to see here"
# use it
```

Reply:
I like your idea a lot, thanks! I am pretty sure `github.com/openshift/origin/pkg/git` package will allow this, I will explore it a bit more